csm_web/frontend/static/frontend/data/enrollment_policies.tsx

```diff
@@ -0,0 +1,294 @@
+        li2_1: "Have no unexcused absences during the first three weeks of section",
+        li2_2: {
+          bullet: "Have no more than two unexcused absences throughout the " + 'semester (see "Guidelines")',
+          subbull_1:
+            "If there are extenuating circumstances that would cause " +
+            "you to miss a section, you must email your mentor with the subject: “",
+          subbull_b: "[Request for Absence] <course>",
+          subbull_2:
+            '” (e.g. "[Request for Absence] CS61A") detailing the ' +
+            "reasons for your absence, and cc mentors@berkeley.edu."
+        },
+        li2_3: {
+          bullet:
+            "Fill out both the mid-semester feedback survey and " +
+            "end-of-semester feedback survey, which are sent via email " +
+            '"(see "Guidelines")',
+          subbull_b: "Even if you are not taking CSM for a unit",
+          subbull_2:
+            ", failure to fill out either feedback form may affect " +
+            "your ability to enroll in CSM sections in future " +
+            "semesters, and may result in you being dropped from your section."
+        }
+      }
+    },
+    faq: {
+      title: "FAQ",
+      body: {
+        p1_b: "What courses does CSM offer sections for?",
+        p2:
+          "We offer small tutoring sections (3-6 students) for CS 61A, " +
+          "CS 61B, CS 61C, CS 70, CS 88, EECS 16A, and EECS 16B. We do " +
+          "not offer 1-1 tutoring.",
+        p3_b: "What requirements are there to join a section?",
+        p4:
+          "You must currently be enrolled in the course you wish to take " +
+          "a section for. If you drop the course, we ask that you drop " +
+          "from the CSM section on Scheduler.",
+        p5_b: "Sections are full. Can I be put on a waitlist?",
+        p6:
+          "Sections are first come first serve, and we do not have a " +
+          "waitlist. We recommend keeping an eye on Scheduler for any " +
+          "spots that may open up if other students drop.",
+        p7_b: "I'm not sure if I should take the unit—what should I be considering?",
+        p8: "Good reasons to enroll in the unit:",
+        li1: "You need extra units to meet your college's requirements or to " + "receive financial aid",
+        li2: "You're looking for a way to hold yourself accountable to " + "attending section",
+        p9_1: "Good reasons to ",
+        p9_b: "not",
+        p9_2: " enroll in the unit:",
+        li3: "You're unsure whether you plan to stay enrolled in CSM, and are " + "just trying it out",
+        li4: "You're concerned about exceeding the unit cap for your college, " + "or exceeding the 33% PNP limit",
+        p10:
+          "It's very possible that none of the above reasons apply to you, " +
+          "in which case you can choose to or to not enroll in the unit " +
+          "however you'd like."
+      }
+    }
+  };
+  
+  export default SECTIONS;
```

Comment:
Include a newline at the end of the file.

Reply:
new line added - til something new about text files